feg/protos/mconfig/mconfigs.proto

```diff
@@ -162,6 +162,7 @@ message AAAConfig {
     // enable event logging for aaa events
     bool EventLoggingEnabled = 5;
     RadiusConfig RadiusConfig = 6;
+    bool AcctReportingEnabled = 7; // enable accounting reporting to the module's orc8r service
```

Comment:
minor nit but I think if you put the comment in the line above, it gets included in the generated files.

Reply:
👍